test/common/testobjects.go

```diff
@@ -88,18 +87,18 @@ func NewTestTargetObject(typeConfig typeconfig.Interface, namespace string, fixt
 	}
 
 	federate.SetBasicMetaFields(obj, typeConfig.GetTarget(), "", namespace, "test-e2e-")
+	if typeConfig.GetTarget().Kind == util.NamespaceKind {
+		// Namespace crud testing needs to have the same name as its namespace.
```

Comment:
Thank you for clarifying that this is intended to create a non-federated resource.  Why, then, would a namespace need to have its namespace set given that is only a requirement for a federated namespace?

Reply:
Got your point, Moved it to the actual test location.